src/mlnet/Telemetry/Events/MLNetCommandEvent.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.DotNet.Cli.Telemetry;
+using Microsoft.ML.CLI.Data;
+using Microsoft.ML.CLI.Utilities;
+
+namespace Microsoft.ML.CLI.Telemetry.Events
+{
+    internal class MLNetCommandEvent
```

Comment:
comment or a more descriptive name would be nice here, something like "AutoTrainCommandInvocationEvent".

Reply:
+1. If I were designing this from scratch, I would not have chosen this name. However, this was the original name chosen for the AutoML invocation event. Please see further description in comment above / possible concern about backwards compatibility